modules/uhttp/filters.go

```diff
@@ -80,6 +81,21 @@ func tracingServerFilter(host service.Host) FilterFunc {
 	}
 }
 
+// authFilter authorizes services based on configuration
+func authFilter(host service.Host) FilterFunc {
+	return func(ctx context.Context, w http.ResponseWriter, r *http.Request, next Handler) {
+		fxctx := &fxcontext.Context{
+			Context: ctx,
+		}
+		authClient := uauth.Client()
+		err := authClient.Authorize(fxctx)
+		if err != nil {
+			fxctx.Logger().Error(uauth.ErrAuthorization, "error", err)
+		}
```

Comment:
hmm. I don't have access to metrics, or host here. I added metrics for authorization.